Rebel Arcade page, header, footer

jskahmeyer opened the pull request.

albertvolkman commented on pages/arcade.liquid:
```diff
@@ -0,0 +1,1304 @@
+          description: This concept is one of the Rebellion's favorite and used by Rebels everywhere. Imagine dropping off a very non-traditional resume in order to be considered as a vendor for the product or service you provide - instead of a lifeless brochure or boring white paper. It's different, it's fun and it works. Use on any contact at the org that you are trying to get in touch with. Wi ...
+          image: "/img/arcade/product-rebel-resume.webp"
+          image-mobile: "/img/arcade/product-rebel-resume-mobile.webp"
+          link: "https://buy.stripe.com/6oE14p7Me5r8ewoeVD"
```
New link- https://buy.stripe.com/28o00l1nQ1aScog5la

albertvolkman commented on _includes/pages/arcade/header.liquid:
```diff
@@ -0,0 +1,177 @@
+<style>
```
For the header and footer, let's implement it site-wide (you can remove the existing header/footer)

jskahmeyer replied:
Just fyi the new header and footer use bootstrap, so I had to update head.liquid to include the bootstrap css/js on all pages.

albertvolkman commented on _includes/pages/arcade/footer.liquid:
```diff
@@ -0,0 +1,168 @@
+              >info@thesalesrebellion.com</a
+            >
+          </div>
+          <div class="d-flex align-items-center">
```
Directly under this could you please add the Prayer Reqeust form? Maybe use a prayer emoji for it. It should link to https://thesalesrebellion.typeform.com/to/JFkwGN7S

jskahmeyer replied:
For the other icons in the footer Ian was using a pixel-icon library that didn't have any good options for a prayer-related icon, so we just went with a simple button for this.

albertvolkman commented on pages/arcade.liquid:
```diff
@@ -0,0 +1,1304 @@
+                      <img src="/img/svg/pixel-heart.svg" alt="Heart">
+                    {% endfor %}
+                  </div>
+                  <h4 class="font-horsemen text-uppercase mb-0">Level {{ parent_index }}</h4>
```
Looking at this with Dale this morning and thinking having Level X here again feels redundant. This might be a good question for Ian if we should simply move the title of the product up to here? Or maybe do something fun with "Easy Mode" "Beginner" etc

jskahmeyer replied:
Ian's still thinking on this, will update PR asap

albertvolkman commented on pages/arcade.liquid:
```diff
@@ -0,0 +1,1304 @@
+                  {% if product.link == '' %}
+                    disabled
+                  {% endif %}
+                  >Buy Now</a
```
We also chatted a bit about this verbiage. Instead of `Buy Now` maybe something like `Unlock` or something similar

albertvolkman commented on pages/arcade.liquid:
```diff
@@ -0,0 +1,1304 @@
+          <div class="level-heading d-none d-md-flex justify-content-start align-items-center">
+            <div>
+              {% for i in (1..parent_index) %}
+                <img src="/img/svg/pixel-heart.svg" alt="heart">
```
These hearts look great! A fun interactive element would be to have the hearts fill up on hover or scroll. Would love to hear what Ian thinks on this.

jskahmeyer replied:
Ian voted against this (his comments are in email thread yesterday)

albertvolkman commented on pages/arcade.liquid:
```diff
@@ -0,0 +1,1304 @@
+        - name: Missing Persons
+          price: 198.00
+          description: How many times in your sales cycles have you been ghosted by your prospect or client? A lot, huh? Well, good news. We made you something them that will make them laugh, potentially cry laugh, but definitely help you get their attention. It's so good that the prospects entire office will get a laugh out of it and you'll get their attention back. With this digital product you ...
+          image: "/img/arcade/product-missing-persons.webp"
```
Oopsie, looks like we're missing this image!

jskahmeyer replied:
Whoops, thanks for catching that!

albertvolkman commented on pages/arcade.liquid:
```diff
@@ -0,0 +1,1304 @@
+          image-mobile: "/img/arcade/product-crumpled-letter-mobile.webp"
+          link: "https://buy.stripe.com/9AQ28td6y2eW880cNq"
+        - name: Rebellious Resume
+          price: 24.00
```
$24.99 now

albertvolkman commented:
The header when implemented site-wide is a bit abrupt (black navigation to the colorful header). What does Ian think about maybe adding a fade at the bottom? And then maybe making the header shrink and be sticky to the top when scrolling down?

albertvolkman commented:
Looks like we might need a bit of work on the header (it doesn't extend the full width of the browser on desktop)-
<img width="1940" alt="Screenshot 2025-04-14 at 4 04 20 PM" src="https://github.com/user-attachments/assets/881da26f-815b-4784-9b3a-90f3ef1bd522" />
I was also hoping we could get a bit of a fade for the header as well, if that's an option.

jskahmeyer commented:
> Looks like we might need a bit of work on the header (it doesn't extend the full width of the browser on desktop)- <img alt="Screenshot 2025-04-14 at 4 04 20 PM" width="1940" src="https://private-user-images.githubusercontent.com/315328/433536472-881da26f-815b-4784-9b3a-90f3ef1bd522.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ2NjQ0MjUsIm5iZiI6MTc0NDY2NDEyNSwicGF0aCI6Ii8zMTUzMjgvNDMzNTM2NDcyLTg4MWRhMjZmLTgxNWItNDc4NC05YjNhLTkwZjNlZjFiZDUyMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQxNFQyMDU1MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTNlNWU4MTEwNDljNjY0OTQ1MmM0MzMyZjEyYTI5ZjU0YWRlYzI2M2QwODNmYWI1ZWIwZDIyOGZmNmE4NDIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.1aLbu2Q6zrvc0WSMK2_KZmJrclkpZjpWplljAiMj-gM"> I was also hoping we could get a bit of a fade for the header as well, if that's an option.

Ack, sorry about that, I made a change to the header so it would still work on the Totality page and introduced that issue. Fixed now. Checking with Ian about the fade, are you picturing it fading to transparent at the bottom of the nav?

albertvolkman commented:
yeah at least until the scroll bar fixes to the top

albertvolkman commented:
BOOM! Love it! Nice work @jskahmeyer !!
